projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/aux-key.e2e-spec.ts

```diff
@@ -184,9 +169,9 @@ context('Auxiliary Keys', () => {
 
 function loadPageWithComponenents(pageUrl: string) {
   cy.server();
-  cy.route(
-    `${Cypress.env('API_URL')}/rest/v2/electronics-spa/cms/components*`
-  ).as('getComponents');
+  cy.route(`${Cypress.env('API_URL')}/rest/v2/electronics-spa/**`).as(
```

Comment:
This url will match a lot of different urls. Not sure if we want to that. The previous solution might've been better